xmtp_mls/src/xmtp_openmls_provider.rs

```diff
@@ -1,21 +1,32 @@
+#[cfg(not(target_arch = "wasm32"))]
+use crate::storage::native::NativeDb;
```

Comment:
Ah, we might as well move `xmtp_open_mls_provider` to be a module of storage, and make wasm/native just public to `super`.

Reply:
Even better